configure.ml

```diff
@@ -512,6 +512,8 @@ let camltag = match caml_version_list with
   | x::y::_ -> "OCAML"^x^y
   | _ -> assert false
 
+let coq_warn_flag =
+  if caml_version_nums >= [4;2;0] then "-w -3-52-56" else ""
```

Comment:
I don't know exactly. If no new warnings appeared in 4.2.0, I guess that 4.3.0 should be ok.